ros2doctor/ros2doctor/api/network.py

```diff
@@ -74,11 +76,10 @@ def check(self):
         if not _is_unix_like_platform():
             if not has_loopback and not has_non_loopback:
                 # no flags found, otherwise one of them should be True.
-                doctor_error(
-                    'No flags found. '
-                    'Run `ipconfig` on Windows or '
-                    'install `ifconfig` on Unix to check network interfaces.')
-                result.add_error()
+                doctor_warn(
+                    'Could not find any available network interfaces.'
```

Comment:
perhaps you were going to change the message again (https://github.com/ros2/ros2cli/pull/631#discussion_r618413015), but just in case :smile: 

```suggestion
                    'Could not find any available network interfaces. '
```

Reply:
thanks!